### PR TITLE
OADP-3522: ROSA docs is missing instructions related to the internal image backup

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-3522](https://issues.redhat.com/browse/OADP-3522)

### PREVIEW

* [OADP ROSA docs](https://72112--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications)

### VERSIONS

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16

### QE review:
- [X ] QE has approved this change. Prasad approved

image

Additional information:

### CODE REVIEW

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/installing-oadp-rosa-sts.adoc [hunk 1]
```diff
@@ -179,6 +180,8 @@ $ cat << EOF | oc create -f -
         enable: false
 EOF
 ----
+<1> Internal image backup is supported with ROSA. Only set this field to false, you want to explicitly skip the image backup.
```
Comment (agantony) [inline modules/installing-oadp-rosa-sts.adoc:183]:
```suggestion
<1> ROSA supports internal image backup. Set this field to `false` if you do not want to use image backup.
```

**Review by agantony — COMMENTED** [review]
Added a small suggestion for your consideration; overall lgtm!

File: modules/cleanup-a-backup-oadp-rosa-sts.adoc [hunk 2]
```diff
@@ -81,13 +88,14 @@ $ aws s3 rm s3://${CLUSTER_NAME}-oadp --recursive
...
 $ aws s3api delete-bucket --bucket ${CLUSTER_NAME}-oadp
 ----
+
 . Detach the policy from the role by running the following command:
 +
 [source,terminal]
 ----
-$ aws iam detach-role-policy --role-name "${ROLE_NAME}" \
-  --policy-arn "${POLICY_ARN}"
+$ aws iam detach-role-policy --role-name "${ROLE_NAME}" \ --policy-arn "${POLICY_ARN}"
```
Comment (mburke5678) [inline modules/cleanup-a-backup-oadp-rosa-sts.adoc:96]: Do you still need the `\` now that the command is on one line? That character is when the command is split across multiple lines, right?

File: modules/installing-oadp-rosa-sts.adoc [hunk 3]
```diff
@@ -192,13 +195,18 @@ $ cat << EOF | oc create -f -
     name: ${CLUSTER_NAME}-dpa
     namespace: openshift-adp
   spec:
+    backupImages: true
```
Comment (mburke5678) [inline modules/installing-oadp-rosa-sts.adoc:198]: Do we need one of these here also?

> <1> ROSA supports internal image backup. Set this field to `false` if you do not want to use image backup.

**Comment by mburke5678** [comment]
@anarnold97 I had a few suggestions that I am hoping you would consider before we merge. Let me know what you think.

File: modules/cleanup-a-backup-oadp-rosa-sts.adoc (hunk 2 above)
Comment (anarnold97) [reply modules/cleanup-a-backup-oadp-rosa-sts.adoc:96]:
```suggestion
$ aws iam detach-role-policy --role-name "${ROLE_NAME}"  --policy-arn "${POLICY_ARN}"
```

File: modules/installing-oadp-rosa-sts.adoc (hunk 3 above)
Comment (anarnold97) [reply modules/installing-oadp-rosa-sts.adoc]: Nice!

**Comment by anarnold97** [comment]
@mburke5678 -thanks muchly, changed, committed, squashed. Good spots

**Pull request merged**